### PR TITLE
Permit non-absolute URI config values

### PR DESCRIPTION
**Pull request by trombka**

### CODE REVIEW

File: engine/util/src/main/scala/pl/touk/nussknacker/engine/util/config/URIExt.scala [hunk 1]
```diff
@@ -0,0 +1,13 @@
+package pl.touk.nussknacker.engine.util.config
+
+import java.io.File
+import java.net.URI
+
+class URIExt(uri: URI) {
+  def withFileSchemeDefault: URI = if (uri.isAbsolute) uri else new File(uri.getSchemeSpecificPart).toURI
+}
+
+object URIExt {
```
Comment (arkadius) [inline engine/util/src/main/scala/pl/touk/nussknacker/engine/util/config/URIExt.scala:10]: You can just use `implicit class` instead: https://docs.scala-lang.org/overviews/core/implicit-classes.html

File: engine/util/src/main/scala/pl/touk/nussknacker/engine/util/config/URIExt.scala [hunk 2]
```diff
@@ -0,0 +1,13 @@
+package pl.touk.nussknacker.engine.util.config
+
+import java.io.File
+import java.net.URI
+
+class URIExt(uri: URI) {
+  def withFileSchemeDefault: URI = if (uri.isAbsolute) uri else new File(uri.getSchemeSpecificPart).toURI
```
Comment (arkadius) [inline engine/util/src/main/scala/pl/touk/nussknacker/engine/util/config/URIExt.scala:7]: Can you add a test for that? What case will work other way after this change? Is it the case when `new URI(value)` throws an exception?

Comment (trombka) [reply engine/util/src/main/scala/pl/touk/nussknacker/engine/util/config/URIExt.scala:7]: Before:
Any URI in a config class would default to a file if no scheme were specified.
After:
The standard Ficus reader is used for a URI in a config class. If you want a default scheme, use the extension method.

File: engine/util/src/main/scala/pl/touk/nussknacker/engine/util/config/URIExt.scala (hunk 1 above)
Comment (trombka) [reply engine/util/src/main/scala/pl/touk/nussknacker/engine/util/config/URIExt.scala]: Changed.

File: engine/util/src/main/scala/pl/touk/nussknacker/engine/util/config/URIExt.scala (hunk 2 above)
Comment (trombka) [reply engine/util/src/main/scala/pl/touk/nussknacker/engine/util/config/URIExt.scala]: I have added a test.

**Review by arkadius — APPROVED** [review]

**Pull request merged**